com.eclipsesource.modelserver.emf/src/main/java/com/eclipsesource/modelserver/emf/common/ModelRepository.java

```diff
@@ -89,12 +93,22 @@ public void removeModel(String modeluri) {
 		return Optional.ofNullable(this.models.get(uri));
 	}
 
+	private void loadAllModelsInWorkspace() {
+		if (!isModelWorkspaceLoaded()) {
+			serverConfiguration.getWorkspaceEntries().forEach(this::loadModel);
+			this.modelWorkspaceLoaded = true;
+		}
+	}
+
+	private boolean isModelWorkspaceLoaded() {
+		return modelWorkspaceLoaded;
+	}
+
 	private URI getWorkspaceUri(String modeluri) {
 		String baseURL = serverConfiguration.getWorkspaceRoot();
 		if (!modeluri.startsWith(baseURL)) {
-			modeluri = baseURL + "/" + modeluri.replaceAll(" ", "");
+			modeluri = baseURL + modeluri.replaceAll(" ", "");
```

Comment:
How do we know that `baseURL` is ending with a slash here?

Reply:
uri.getPath() (in ServerConfiguration.java) adds a trailing slash to the path. Could not find an actual proof, but I made various observations...

Reply:
Ok, let's track that as a separate issue then.

Reply:
Eugen reported issues that are probably related to this.

ERROR ResourceManager  - Could not load resource with URI: /home/eugen/Git/coffee-editor/backend/examples/SuperBrewer3000SuperBrewer3000.coffee

Seems like the last / is missing